crawler/mesos.py

```diff
@@ -0,0 +1,237 @@
+#! /usr/bin/python
```

Comment:
Ah, yes, please remove it. I'm in the process of doing that actually. Deleting all the dead code. I left a bunch of "just in case" code that was never used.